mediapipe/calculators/ovms/BUILD

```diff
@@ -75,10 +75,12 @@ cc_library(
         "openvinoinferencecalculatoroptions.h"
     ],
     deps = [
+        "openvinomodelserversessioncalculator",
```

Comment:
Why options for inference calculator depend on session calculator?

Reply:
We need access to global log variable

Reply:
Now its just to get env variable name value.

Reply:
Move StringToLogLevel and LogLevelToString to eg. ovms_utils.(cpp|hpp) to break this dependency.

Reply:
It only needs the OpenVINOModelServerSessionCalculator::OvmsLogLevelEnv